lua/autorun/client/cl_ctc_handler.lua

```diff
@@ -205,7 +205,11 @@ function CTC:RunCommand(command)
 	if not self.data.commands[command].console then return end
 
 	for _, line in ipairs(self.data.commands[command].console) do
+		if type(line) == "string" then
 		RunConsoleCommand(line)
+		return end
```

Comment:
unpack works with one argument. I will test transforming the line into a table

Reply:
I rather meant if the table is only 1 long and you you as second argument a 2.

Reply:
Alright, I misunderstood it then. That is what I'm worried about transforming the line into a table too.

Reply:
Transforming the line into a table if it is a string is ideal, as it will only be done the first time the command is executed and `unpack(line, 2)` will, if line[2] etc. are empty, return an empty vararg. RunConsoleCommand accepts this, so there is no need for a special "0 arguments" case. 
I tested with 0, 1 and 3 arguments. All work correctly.